src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -134,17 +134,11 @@ class BankAccountStep extends React.Component {
             return;
         }
 
-        BankAccounts.setupWithdrawalAccount({
-            acceptTerms: this.state.hasAcceptedTerms,
-            accountNumber: this.state.accountNumber,
-            routingNumber: this.state.routingNumber,
-            setupType: CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL,
-
-            // Note: These are hardcoded as we're not supporting AU bank accounts for the free plan
-            country: CONST.COUNTRY.US,
-            currency: CONST.CURRENCY.USD,
-            fieldsType: CONST.BANK_ACCOUNT.FIELDS_TYPE.LOCAL,
```

Comment:
Could you confirm if we still need these?

Reply:
My thought was: If these are hardcoded, why pass them as parameters?

Reply:
I don't think we're hardcoding them on the API side, though. So I think we'll need to add this hardcoding in PHP somewhere?

Reply:
I don't think so? At least not for this flow, since we aren't really adding anything to the DB